src/sentry/tasks/post_process.py

```diff
@@ -595,6 +595,37 @@ def process_rules(job: PostProcessJob) -> None:
     return
 
 
+def process_code_mappings(job: PostProcessJob) -> None:
+    if job["is_reprocessed"]:
+        return
+
+    from sentry.tasks.derive_code_mappings import derive_code_mappings
+
+    try:
+        event = job["event"]
```

Comment:
Are the stacktraces available at this point? if all stacktraces match a code mapping we will not need to schedule anything.

Reply:
We should be able to get stacktraces here like [this](https://github.com/getsentry/sentry/blob/c1b7994345096b15efff054341ce569dea57e76b/src/sentry/tasks/derive_code_mappings.py#L110). How would we check that it matches a code mapping?

Reply:
Thinking this more. We should not put the logic here, otherwise, we hit the DB for every event.